internal/test/wsu/wsu_test.go

```diff
@@ -100,15 +96,15 @@ func setupAndTest(t *testing.T, vmNum int) {
 	t.Parallel()
 
 	// Run the WSU against the VM
-	wsuOut, err := runWSU(framework.WinVMs[vmNum])
+	wsuOut, err := runWSU(framework.WinVMs[vmNum], vmNum)
 	require.NoError(t, err, "WSU playbook returned error: %s, with output: %s", err, wsuOut)
 
 	// Run our VM test suite
-	runTest(t, framework.WinVMs[vmNum], wsuOut)
+	runTest(t, framework.WinVMs[vmNum], vmNum, wsuOut)
 }
 
 // runWSU runs the WSU playbook against a VM. Returns WSU stdout
-func runWSU(vm e2ef.WindowsVM) (string, error) {
+func runWSU(vm e2ef.WindowsVM, vmNum int) (string, error) {
```

Comment:
If you have VM struct to additional field called `testLocalBuild`, you wouldn't need to pass vmNum as an argument to any function.

Reply:
I'd argue a more general field than `testLocalBuild` should be used

Reply:
For now I have used `buildWmcb` since that is the flag we are passing through wsu. I am looking for suggestions for a more generalized name